date: 12/02/2021 20:47:50, exception type:<class 'ValueError'>, Message: , 
traceback: <traceback object at 0x0000025DF9F8C780>
date: 12/02/2021 20:48:17, exception type:<class 'ValueError'>, Message: ValueError(), 
traceback: <traceback object at 0x00000194B2339880>
date: 12/02/2021 20:49:38, exception type:ValueError(), Message: , 
traceback: <traceback object at 0x00000179A3C4C800>
date: 12/02/2021 20:49:56, exception type:UnableToWorkException(), Message: I'm not hired yet, lol., 
traceback: <traceback object at 0x00000253B50ED740>
